english/punch-snape/punch-snape.md

```diff
@@ -0,0 +1 @@
+Hermione takes complete leave of her senses in the face of such blatant rule-breaking and punches Professor Snape in the face, Malfoy-in-third-year style.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.